.github/workflows/create-release-post.yml

```
@@ -0,0 +1,54 @@
+# Security Notes
+# Only selected Actions are allowed within this repository. Please refer to (https://github.com/nodejs/nodejs.org/settings/actions)
+# for the full list of available actions. If you want to add a new one, please reach out a maintainer with Admin permissions.
+# REVIEWERS, please always double-check security practices before merging a PR that contains Workflow changes!!
+# AUTHORS, please only use actions with explicit SHA references, and avoid using `@master` or `@main` references or `@version` tags.
+
+name: Create Release Blog Post
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: The version to generate a blog post for. If none provided, use the latest release.
+        type: string
```

```suggestion
        type: string
        required: true
```

Let's always request a version string.

---

Why? We have the ability to infer it from the latest version

---

If the goal is to have this workflow called by another workflow, it’s indeed a bit moot to keep this thing optional. Let’s make it mandatory and simplify the implementation

---

In https://github.com/nodejs/node/blob/main/.github/workflows/update-release-links.yml#L18, we don't pass the version to that workflow, I would assume it wouldn't be passed to this one.

---

> I would assume it wouldn't be passed to this one.

I would not assume that, that would be a very suboptimal decision